code/ATMOSPHERICS/pipes.dm

```diff
@@ -604,12 +600,8 @@
 	color = PIPE_COLOR_GREEN
 /obj/machinery/atmospherics/pipe/manifold/insulated
 	name = "\improper Insulated pipe"
-	//icon = 'icons/obj/atmospherics/red_pipe.dmi'
-	icon_state = "manifold"
```

Comment:
This is the only line in this PR that does what you say it does. The rest is not only non atomic but it'll cause bugs as well - for example, I'm pretty sure, and don't quote me on this, but you won't be able to colour insulated pipes using the RPD.